doc/notebooks/Dask.ipynb

```diff
@@ -445,17 +445,17 @@
        "    title:        4x daily NMC reanalysis (1948)\n",
```

Comment:
<p>Can we use an existing dataset to simplify this step ? That is, reduce the boilerplate code not directly relevant to what you want to demonstrate. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/xESMF/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>